Libraries/src/Amazon.Lambda.PowerShellHost/PowerShellFunctionHost.cs

```diff
@@ -166,12 +166,22 @@ private IAsyncResult BeginInvoke(string input, ILambdaContext context)
 
 ";
 
-            var tempFolder = string.IsNullOrEmpty(Environment.GetEnvironmentVariable("LAMBDA_TASK_ROOT")) ? "/tmp" : Path.GetTempPath();
+            var isLambda = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("LAMBDA_TASK_ROOT"));
+
+            var tempFolder = isLambda ? "/tmp" : Path.GetTempPath();
```

Comment:
Is the ! correct on line 169? This is flipping /tmp and Path.GetTempPath() from what the code used to do when it is used on 171.

Reply:
It is correct. The old behavior was wrong. I added a new test to cover this missing area in the test coverage.